Python/PokerGameController.py

```diff
@@ -325,8 +322,7 @@ def debug(self) -> None:
     model = PokerGameModel()
     view = PokerGameView(model)
     turret = Turret("/dev/tty.Bluetooth-Incoming-Port")
-    shuffler = Shuffler("/dev/tty.Bluetooth-Incoming-Port")
-    controller = PokerGameController(model, view, turret, shuffler)
+    controller = PokerGameController(model, view, turret)
```

Comment:
We're removing the shuffler from the Controller? Just commenting so this doesn't go unnoticed even if intentional